src/alfasim_sdk/_internal/alfacase/generate_case_description_docstring.py

```diff
@@ -169,7 +169,10 @@ def _get_declaration(
 
                     # For Schema: if the attributes have a default value the entry is optional.
                     if default_value and is_schema:
-                        attribute_value += " # optional"
+                        if attribute_value.startswith("\n"):
+                            attribute_value = "# optional" + attribute_value
+                        else:
+                            attribute_value += " # optional"
```

Comment:
```suggestion
                        if attribute_value.startswith("\n"):
                            attribute_value = " # optional" + attribute_value
                        else:
                            attribute_value += "  # optional"
```
Just to avoid the `# optional` comment "touching" the attribute try to add a space here...